CDC-Data-Reconciliation-Frontend/src/components/Report.jsx

```diff
@@ -37,6 +58,69 @@ export default function Report({ reportID }) {
             <h3>Number of Cases Different: {results.length}</h3>
           </div>
 
+		  {/* Report and Disease Specific Stats */}
+		  <div>
+			{statistics && (
+			<>
+			<button
+				type="button"
+				className="bg-blue-400 text-white px-5 py-2 rounded-md hover:bg-blue-600 mb-4"
+				onClick={toggleDiseaseStats}
+			>
+				{showDiseaseSpecificStats ? "Hide" : "Show"} Disease Specific Stats
+			</button>
+
+			{!showDiseaseSpecificStats ? (
+				<table className="w-full text-center mb-4">
+				<thead>
+					<tr className="border-b-2 border-slate-900">
+					<th>Total Cases</th>
+					<th>Total Duplicates</th>
+					<th>Total Missing From CDC</th>
+					<th>Total Missing From States</th>
+					<th>Total Wrong Attributes</th>
+					</tr>
+				</thead>
+				<tbody>
+					<tr>
+					<td>{statistics.TotalCases}</td>
+					<td>{statistics.TotalDuplicates}</td>
+					<td>{statistics.TotalMissingFromCDC}</td>
+					<td>{statistics.TotalMissingFromStates}</td>
+					<td>{statistics.TotalWrongAttributes}</td>
+					</tr>
+				</tbody>
+				</table>
+			) : (
+				<table className="w-full text-center">
+				<thead>
+					<tr className="border-b-2 border-slate-900">
+					<th>Event Code</th>
+					<th>Total Cases</th>
+					<th>Total Duplicates</th>
+					<th>Total Missing From CDC</th>
+					<th>Total Missing From States</th>
+					<th>Total Wrong Attributes</th>
+					</tr>
+				</thead>
+				<tbody>
+					{statistics.DiseaseStats.map((stat) => (
```

Comment:
There does not exist a DiseaseStats attribute on the statistics data you get back from the API. you should just use statistics.map